torch/onnx/symbolic_opset9.py

```diff
@@ -4961,7 +4964,26 @@ def ListConstruct(g, *inputs, **kwargs):
         return None
 
     @staticmethod
-    def ListUnpack(g, *inputs, **kwargs):
+    def ListUnpack(g, *inputs, **kwargs) -> Optional[List[_C.Value]]:
+        """Translates the ListUnpack Op.
+
+        Args:
+            g: The current graph.
+            *inputs: The inputs to the ListUnpack Op.
+            **kwargs: Additional keyword arguments.
+
+        Returns:
+            None if unchanged. Inputs to its previous op if the previous op is
+            ListConstruct.
+        """
+        if not inputs:
+            return None
+        first_input = inputs[0]
```

Comment:
`inputs[0]` is shorter and just as clear as `first_input`, so I'd get rid of this variable.

Reply:
Done